pages/reference-files/[id].js

```diff
@@ -48,35 +55,52 @@ export default function ReferenceFile({
           <FileHeaderDownload file={referenceFile} />
         </ObjectPageHeader>
         <JsonDisplay item={referenceFile} isJsonFormat={isJson}>
-          <DataPanel>
-            <DataArea>
-              <FileDataItems
-                item={referenceFile}
-                fileSet={fileSet}
+          <FileDataItems item={referenceFile} fileSet={fileSet}></FileDataItems>
+          {("assembly" in referenceFile || "source_url" in referenceFile) && (
+            <>
+              <DataAreaTitle>Reference Details</DataAreaTitle>
+              <DataPanel>
+                <DataArea>
+                  {referenceFile.assembly && (
+                    <>
+                      <DataItemLabel>Genome Assembly</DataItemLabel>
+                      <DataItemValue>{referenceFile.assembly}</DataItemValue>
+                    </>
+                  )}
+                  {referenceFile.source_url && (
+                    <>
+                      <DataItemLabel>Source URL</DataItemLabel>
+                      <DataItemValue>
+                        <Link
+                          href={referenceFile.source_url}
+                          key={referenceFile.source_url}
+                        >
+                          {referenceFile.source_url}
+                        </Link>
+                      </DataItemValue>
+                    </>
+                  )}
+                </DataArea>
+              </DataPanel>
+            </>
+          )}
+          {derivedFrom?.length > 0 && (
+            <>
+              <DataAreaTitle>
+                Files {referenceFile.accession} derives from
```

Comment:
Use title case for the title.

Reply:
ENCODE did not which I based the table off, but I'll change it.

Reply:
Yeah, ENCODE started with sentence case, but so many people made new tables, menus, and other items and mistakenly used title case, that I figured it’s better to just keep to title case in igvf-ui to try to reduce the mistakes.